tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -0,0 +1,472 @@
+            throw error;
+        }
+        return result;
+    }
+
+    public static ArrayList<String> getAllTSSTags(ThresholdKey thresholdKey) throws RuntimeError, JSONException {
+        RuntimeError error = new RuntimeError();
+        String result = jniTSSModuleGetAllTSSTags(thresholdKey, error);
+        if (error.code != 0) {
+            throw error;
+        }
+        ArrayList<String> tssTags = new ArrayList<>();
+        JSONArray jsonArray = new JSONArray(result);
+
+        for (int i = 0; i < jsonArray.length(); i++) {
+            tssTags.add(jsonArray.getString(i));
+        }
+        return tssTags;
+    }
+
+    public static ArrayList<String> getAllFactorPub(ThresholdKey thresholdKey) throws RuntimeError, JSONException {
+        RuntimeError error = new RuntimeError();
+        String result = jniTSSTagFactorPub(thresholdKey, error);
+        if (error.code != 0) {
+            throw error;
+        }
+        ArrayList<String> factorPubs = new ArrayList<>();
+        JSONArray jsonArray = new JSONArray(result);
+
+        for (int i = 0; i < jsonArray.length(); i++) {
+            factorPubs.add(jsonArray.getString(i));
+        }
+        return factorPubs;
+    }
+
+    public static String getExtendedVerifier(ThresholdKey thresholdKey) throws RuntimeError, JSONException {
+        RuntimeError error = new RuntimeError();
+        String result = jniGetExtendedVerifier(thresholdKey, error);
+        if (error.code != 0) {
+            throw error;
+        }
+        return result;
+    }
+
+    public static void setTSSTag(ThresholdKey thresholdKey, String tssTag, ThresholdKeyCallback<Boolean> callback) {
+        thresholdKey.executor.execute(() -> {
+            try {
+                Result<Boolean> result = setTSSTag(thresholdKey, tssTag);
+                callback.onComplete(result);
+            } catch (Exception e) {
+                Result<Boolean> error = new Result.Error<>(e);
+                callback.onComplete(error);
+            }
+        });
+    }
+
+    private static Result<Boolean> setTSSTag(ThresholdKey thresholdKey, String tssTag) {
+        try {
+            RuntimeError error = new RuntimeError();
+            jniTSSModuleSetTSSTag(thresholdKey, tssTag, error);
+            if (error.code != 0) {
+                throw new Exception(error);
+            }
+            return new Result.Success<>(true);
+        } catch (Exception e) {
+            return new Result.Error<>(e);
+        }
+    }
+
+    public static String getTSSTag(ThresholdKey thresholdKey) throws RuntimeError {
+        RuntimeError error = new RuntimeError();
+        String result = jniTSSModuleGetTSSTag(thresholdKey, error);
+        if (error.code != 0) {
+            throw error;
+        }
+        return result;
+    }
+
+    public static GetTSSPubKeyResult getTssPubAddress(ThresholdKey thresholdKey, String tssTag, String nonce, NodeDetails nodeDetails, TorusUtils torusUtils) throws Exception, RuntimeError {
```

Comment:
This is getting dkg tss pub key. 
there is another tss pub key get method which return the final tss pub key from rust lib